### DISCUSSION TITLE
quick little bypass  os127

### DISCUSSION THREAD

**Discussion by Solar-Geaux** [3kh0/ext-remover#1174] [Webview JOIN TITANIUM NETWORK TO SHARE PLEASE]
WHAT YOU CAN DO WITH THIS: UNRESTRICTED BROWSING, TEACHERS CANT SEE SCREEN ETC, EXPLOIT STAYS WORKING EVEN AFTER CLOSING SCREEN.
NEGATIVES: your kinda stuck on full screen, of only one tab at a time, you can only go to different websites using the Google home page or some other search engine if you can get to it, USE THE 3kh0 REDIRECTOR to go to specific websites Google doesn't display. logging into any Gmail account will make the exploit unusable and you'll have to power wash to be able to do it again. going back and forth between pages you have to use the arrow keys next to escape on your keyboard.
SOME PAGES LIKE GITHUB, YOUTUBE AND MANY OTHERS MAY BE blocked (since every proxy is now unblocked just use one of those..) the websites aren't being blocked by an extension, rather the organization(when logged in you can probably go on the pages, but there's some weird thing with it not allowing you to idrk)

I tested this on version 127 of chrome os,this exploit only works if you have the 'app' fastbridge learning. (I've tried 6 other apps my school gave to me, none worked besides fastbridge learning)

1.  first log out of your Chromebook, 
2. click the fastbridge learning application in the bottom left in the 'apps category'
3. click the button in the top right, (I forgot what its called but its to the right of 'check status') check status used to work as the exploit however it was patched and It took longer.
4. you'll see an about us tab in the list, hover over it and click the 'leadership' button
5. scroll down and you'll see a 'blackstone.com' button click that
6. once your on this page type in Google on the search bar and 'cookie policy' will show up in the list
7. click that and then scroll down to the contact area, you'll see the Email: PrivacyQueries@gmail.com
8. click that link and you'll be sent to a Google page, click the resources tab and click blog (no other Google pages work, somehow only blog works with this) 
9. scroll down all the way to the bottom, in the bottom left a google logo will be shown, click that and you'll be sent to the regular Google search page, now you can go on any website etc, 
I RECOMMEND USING A PROXY THAT ALLOWS MULTI TABS SINCE YOU CANT HAVE MULTIPLE TABS IN THE BYPASS BY ITSELF. 

if you read all this and don't have the app, you can try yourself to do this on any apps you may have, I was bored one day in geometry class and was in lockdown mode on my browser (cant go on websites) so I decided to find a bypass by clicking around quite a lot (if this method doesn't work, feel free to ask for more, I found 3 other methods but they take much longer, nonetheless they work.)

**Comment by Solar-Geaux** [comment]
if your wondering why I didn't click any other link or email its because they simply don't work or send you to an email writing page where you get stuck on so yeah this is the only way it works.

**Comment by S-PScripts** [comment]
"use a **_proxy_** for multi tabs"

hmm....

**Reply by funkymokey** [reply]
what could this mean

**Reply by Solar-Geaux** [reply]
thing is many good proxies are blocked on regular os, not to mention the best part of the whole bypass is that teachers cant see your screen, delete tabs, freeze screen, turn off wifi etc.

**Comment by akabutnicer** [comment]
So just a webview I presume? This isn't only bound to v127, you know that right?

**Reply by Solar-Geaux** [reply]
pretty much, redirects where the page opens in a new tab wont work. I tested it in v127 and so that's all the data I have for it, it hasn't been patched for a year but it COULD be, I don't know if it has since I'm on v127.

**Reply by akabutnicer** [reply]
Fair enough. Although, due to the `<webview />` HTML tag (the main component of this exploit) being granted to these Chrome apps (including kiosk ones), only the app developer can prevent this exploit. It would be irrational for Chrome to interpolate themselves on this. Not saying this couldn't happen though.
